### ISSUE TITLE
dfDropdown.popup null test

### ISSUE THREAD

**Issue by GreeneDragon** [TakuanDaikon/DFGUI#9]
hi, I've noticed that dfDropdown.popup always evaluates as null. Therefore, clicking on the button always opens the popup rather than toggle it on/off. The cause for this remains a mystery to me.

I haven't updated my DFGUI since it became open source. I am wondering if that issue has been fixed?

**Comment by TakuanDaikon** [comment]
I don't believe that it has been fixed. I haven't had any time to look at DFGUI at all lately, sorry.

**Comment by GreeneDragon** [comment]
Thanks for replying. I am still a loyal fan of DFGUI. I learned a lot from
you!

Jason

**Comment by kfc79** [comment]
Thanks for the fixes Greene Dragon.
I’m a fan of DFGUI too.
Really glad to see an update here.


Cheers,
KFC
